packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
This happens a lot when building on a Mac. Just revert it.

Reply:
@tylerbutler, It is not limited to Macs. I see this all the time on Windows as well.

Reply:
Let's start a teams thread and see if we can collectively root cause it.

Reply:
Might be an api-extractor issue that we should include on the list of things to talk to them about. (I think @Josmithr has a few.)